Functions/xASL_adm_CleanUpBeforeRerun.m

```diff
@@ -367,7 +367,7 @@ function xASL_adm_CleanUpBeforeRerun(AnalysisDir, iModule, bRemoveWMH, bAllSubje
 
                     % Always remove previous paths, irrespective of iModule
                     PathFields = {'MyPath', 'P', 'D', ...
-                        'SPMDIR', 'SPMpath', 'STRUCT_TEMPLATE_IM', 'LOCKDIR', 'SUBJECTDIR',...
```

Comment:
Good that you removed the SPMDIR and SPMpath there, but should you add removal of these fields from the subfields?

Reply:
I don't think so. x.D is removed there and if SPMDIR and SPMpath are in D, then they will be removed as well?

Reply:
Of course - you are right :) I didn't realize that...